src/content/docs/agents/net-agent/attributes/enable-disable-attributes-net.mdx

```diff
@@ -11,7 +11,7 @@ redirects:
   - /docs/agents/net-agent/attributes/enabling-disabling-attributes-net
 ---
 
-This describes the rules New Relic uses to determine which attributes to include or exclude for a [destination](/docs/features/agent-attributes#destinations). This also includes a summary of the .NET agent properties that were deprecated with the release of New Relic agent attributes in versions 3.6.177.0 or higher.
+This describes the rules New Relic uses to determine which attributes to include or exclude for a [destination](/docs/features/agent-attributes#destinations). This also includes a summary of the .NET agent properties that were obsolete with the release of New Relic agent attributes in versions 9.0 or higher.
```

Comment:
My understanding of how industry using the word "deprecated" and "obsolete" is that:

Deprecated means that it is still in use, but only for historical purposes and it will be removed probably in the next big release. Obsolete means that is already out-of-use.

The public apis and configurations that marked as "obsolete" are ones that no longer available in v9.0.

Reply:
@vuqtran88, OK, you can ignore my comments about keeping "deprecated," but you'll still need to avoid using obsolete as a verb. So, a safe choice is to say something like "The X configuration is obsolete in version 9." Hope that helps! Sorry for the confusion on my end!

Reply:
I think for this message, it probably makes more sense to change from "obsolete" to "not available".

Reply:
@vuqtran88, I pushed up some changes for this. Let me know if they look OK.